components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2084,14 +2077,8 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     public async getTeamMembers(ctx: TraceContext, teamId: string): Promise<TeamMemberInfo[]> {
         traceAPIParams(ctx, { teamId });
 
-        if (!uuidValidate(teamId)) {
-            throw new ResponseError(ErrorCodes.BAD_REQUEST, "organization ID must be a valid UUID");
-        }
+        const { members } = await this.guardTeamOperation(teamId, "get");
```

Comment:
`this.checkUser("getTeamMembers");` got MIA here.

Reply:
pushed a fix for this